docs/how-to/companion-node-types.md

```diff
@@ -67,6 +67,12 @@ This node type offers the same benefits as an [external](#external) node, with a
 
 ## Embedded
 
+::: warning DEPRECATED
+
+js-ipfs has been deprecated in favor of [Helia](https://github.com/ipfs/helia) and will be removed in a future release of IPFS Companion. Improvements to Helia are ongoing, and we hope to have it ready for use in IPFS Companion once critical features like [webRTC Streams](https://github.com/w3c/webextensions/issues/72) are available in Chrome Manifest V3.
+
+:::
```

Comment:
@whizzzkid : I think this just adds noise.  This can be in our backlogged but this was a non-functioning feature.  I think we should just strip it out.